pylint/test/unittest_reporters_json.py

```diff
@@ -34,6 +34,7 @@ def test_simple_json_output():
         ("column", 0),
         ("line", 1),
         ("message", "Line too long (1/2)"),
+        ("message-id", "C0301"),
```

Comment:
And what about adding the message-id (C0301 for example) and the message-symbol (line-too-long)?